d2-int-test/src/test/java/com/linkedin/d2/loadbalancer/strategies/TestLoadBalancerStrategy.java

```diff
@@ -0,0 +1,116 @@
+package com.linkedin.d2.loadbalancer.strategies;
+
+import com.linkedin.d2.balancer.strategies.framework.LatencyQPSCorrelation;
+import com.linkedin.d2.balancer.strategies.framework.LoadBalancerStrategyTestRunner;
+import com.linkedin.d2.balancer.strategies.framework.LoadBalancerStrategyTestRunnerBuilder;
+import com.linkedin.d2.loadBalancerStrategyType;
+import java.net.URI;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import org.testng.annotations.Test;
+
+import static com.linkedin.d2.balancer.properties.PropertyKeys.*;
+import static org.testng.Assert.*;
+
+
+/**
+ * Integration tests for all the load balancer strategies
+ */
+public class TestLoadBalancerStrategy {
+  private static final String DUMMY_SERVICE_NAME = "dummyService";
+
+  @Test
+  public void testStrategyV3HealthPoints() {
+    long healthyLatency = 10L;
+    long unhealthyLatency = 3100L; // For strategyV3, the default highLatency is 3000
+
+    LoadBalancerStrategyTestRunner testRunner = new LoadBalancerStrategyTestRunnerBuilder(loadBalancerStrategyType.DEGRADER, null, DUMMY_SERVICE_NAME, 5)
+        .addStrategyProperty(HTTP_LB_CONSISTENT_HASH_ALGORITHM, "pointBased")
```

Comment:
reference a constant instead of string

Reply:
I changed the tests using the default test scenarios